flocker/provision/_install.py

```diff
@@ -605,9 +613,46 @@ def configure_zfs(node, variants):
     ])
 
 
+def _uninstall_flocker_ubuntu1404():
+    # TODO Does a real upgrade (instead of uninstall/install) work?
+    return run_from_args([
+        b"apt-get", b"remove", b"-y", b"--purge", b"clusterhq-python-flocker",
```

Comment:
I'd definitely like it to clean that stuff up - but I don't think it actually will.  I think ideally the upgrade feature of `run-acceptance-tests` would just be a faster way to get a clean node (possibly I should make the docs for it explicitly state that).  For now I think the combination of the package upgrade, process restart, and the acceptance tests state cleanup code produces something working-enough but it could probably be better.